File: 品味Java的21種設計模式/dp_cpp/adapter/example2/LogModel.cpp
#include "LogModel.h"

namespace cn
{
	namespace javass
	{
		namespace dp
		{
			namespace adapter
			{
				namespace example2
				{

					std::string LogModel::getLogId()
					{
						return logId;
					}

					void LogModel::setLogId(std::string logId)
					{
						this->logId = logId;
					}

					std::string LogModel::getOperateUser()
					{
						return operateUser;
					}

					void LogModel::setOperateUser(std::string operateUser)
					{
						this->operateUser = operateUser;
					}

					std::string LogModel::getOperateTime()
					{
						return operateTime;
					}

					void LogModel::setOperateTime(std::string operateTime)
					{
						this->operateTime = operateTime;
					}

					std::string LogModel::getLogContent()
					{
						return logContent;
					}

					void LogModel::setLogContent(std::string logContent)
					{
						this->logContent = logContent;
					}

					std::string LogModel::ToString()
					{
						return "logId="+logId+",operateUser="+operateUser+",operateTime="+operateTime+",logContent="+logContent;
					}
				}
			}
		}
	}
}

File: 品味Java的21種設計模式/dp_java/adapter/example5/LogModel.java
package cn.javass.dp.adapter.example5;

import java.io.*;

/**
 * 日志数据对象
 */
public class LogModel implements Serializable {
    /**
     * 日志编号
     */
    private String logId;
    /**
     * 操作人员
     */
    private String operateUser;
    /**
     * 操作时间，以yyyy-MM-dd HH:mm:ss的格式记录
     */
    private String operateTime;
    /**
     * 日志内容
     */
    private String logContent;

    public String getLogId() {
        return logId;
    }
    public void setLogId(String logId) {
        this.logId = logId;
    }
    public String getOperateUser() {
        return operateUser;
    }
    public void setOperateUser(String operateUser) {
        this.operateUser = operateUser;
    }
    public String getOperateTime() {
        return operateTime;
    }
    public void setOperateTime(String operateTime) {
        this.operateTime = operateTime;
    }
    public String getLogContent() {
        return logContent;
    }
    public void setLogContent(String logContent) {
        this.logContent = logContent;
    }

    public String toString() {
        return "logId=" + logId + ",operateUser=" + operateUser + ",operateTime" + operateTime + ",logContent=" + logContent;
    }
}
